src/watch/stack_trace.rs

```diff
@@ -76,7 +76,7 @@ fn instance_for_call_operand<'a, 'tcx: 'a>(
             }
             ty::FnDef(def_id, substs) => {
                 let substs = ecx.tcx.subst_and_normalize_erasing_regions(
-                    ecx.substs(),
+                    ecx.stack().last().expect("top frame").instance.substs,
```

Comment:
Won't this panic if we look at memory after the program finishes evaluation?

Reply:
That won't happen, once the final return terminator is reached stepping will stop, so there is always at least one stack frame:

https://github.com/bjorn3/priroda/blob/0aec7250ab077a82d678ead19c367ab23092ff5b/src/step.rs#L95